core/backend/src/IModelDb.ts

```diff
@@ -1905,9 +1905,9 @@ export namespace IModelDb { // eslint-disable-line no-redeclare
      * @param aspectProps The properties of the new ElementAspect.
```

Comment:
Document the new return type please.

Reply:
I assume you mean in NextVersion.md, will do

Reply:
Maybe, but I mean right here so readers of the API doc know that it `@returns the Id of the newly-inserted aspect` or whatever.

Reply:
done